src/Illuminate/Database/Connection.php

```diff
@@ -391,11 +391,12 @@ public function selectFromWriteConnection($query, $bindings = [])
      * @param  string  $query
      * @param  array  $bindings
      * @param  bool  $useReadPdo
+     * @param  array  $fetchUsing
```

Comment:
shouldn't this be
```suggestion
     * @param  int-mask-of<PDO::FETCH_*>[]  $fetchUsing
```

Reply:
Yes that would be more accurate in certain scenarios, however there are multiple types of arguments as seen here:

- https://www.php.net/manual/en/pdostatement.fetchall.php
- https://www.php.net/manual/en/pdostatement.fetch.php

I dont see how all these cases can be added easily without a long line of typehints.